From 7b03afe68ea58cb979f1e78badd5640744c1ed3b Mon Sep 17 00:00:00 2001
From: razorloves <razorloves@gmail.com>
Date: Sun, 22 Jul 2018 21:19:09 -0500
Subject: [PATCH 211/249] wiki: Update hiae pages

 * Remove hiaeul and hiaeuhl pages. They were unified into hiae.
 * Correct the codename in hiae yaml.
 * Mark hiae as discontinued. Nightlies were stopped a month ago.

Change-Id: Idb71e62a17c13dde53de61b85c1b81f016e01d23
---
 _data/devices/hiae.yml    |  6 ++---
 _data/devices/hiaeuhl.yml | 46 ---------------------------------------
 _data/devices/hiaeul.yml  | 46 ---------------------------------------
 pages/build/hiaeuhl.md    |  8 -------
 pages/build/hiaeul.md     |  8 -------
 pages/info/hiaeuhl.md     |  9 --------
 pages/info/hiaeul.md      |  9 --------
 pages/install/hiaeuhl.md  |  8 -------
 pages/install/hiaeul.md   |  8 -------
 9 files changed, 3 insertions(+), 145 deletions(-)
 delete mode 100644 _data/devices/hiaeuhl.yml
 delete mode 100644 _data/devices/hiaeul.yml
 delete mode 100644 pages/build/hiaeuhl.md
 delete mode 100644 pages/build/hiaeul.md
 delete mode 100644 pages/info/hiaeuhl.md
 delete mode 100644 pages/info/hiaeul.md
 delete mode 100644 pages/install/hiaeuhl.md
 delete mode 100644 pages/install/hiaeul.md

diff --git a/_data/devices/hiae.yml b/_data/devices/hiae.yml
index d1d1266..95ade9b 100644
--- a/_data/devices/hiae.yml
+++ b/_data/devices/hiae.yml
@@ -5,8 +5,8 @@ cameras:
 - {flash: 'Dual-LED', info: '13 MP'}
 - {flash: '', info: '4 MP'}
 carrier:
-channels: [weekly]
-codename: hiaeul
+channels: [discontinued]
+codename: hiae
 cpu: Cortex-A53
 cpu_cores: '8'
 cpu_freq: 4 x 1.5 GHz & 4 x 1.2 GHz
@@ -18,7 +18,7 @@ height: 145.8 mm (5.74 in)
 image: hiae.png
 install_method: fastboot_htc
 kernel: android_kernel_htc_msm8952
-maintainers: [intervigil]
+maintainers: []
 models: [2PQ9120, 2PQ9100]
 name: One A9
 network:
diff --git a/_data/devices/hiaeuhl.yml b/_data/devices/hiaeuhl.yml
deleted file mode 100644
index 7063733..0000000
--- a/_data/devices/hiaeuhl.yml
+++ /dev/null
@@ -1,46 +0,0 @@
-architecture: arm64
-battery: {removable: False, capacity: 2150, tech: 'Li-Ion'}
-bluetooth: {spec: '4.1', profiles: [A2DP]}
-cameras:
-- {flash: 'Dual-LED', info: '13 MP'}
-- {flash: '', info: '4 MP'}
-carrier:
-channels: [weekly]
-codename: hiaeuhl
-cpu: Cortex-A53
-cpu_cores: '8'
-cpu_freq: 4 x 1.5 GHz & 4 x 1.2 GHz
-current_branch: 14.1
-depth: 7.3 mm (0.29 in)
-download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
-gpu: Adreno 405
-height: 145.8 mm (5.74 in)
-image: hiae.png
-install_method: fastboot_htc
-kernel: android_kernel_htc_msm8952
-maintainers: [intervigil]
-models: [2PQ9100]
-name: One A9 (GSM International)
-network:
-- {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
-- {tech: 3G, bands: '850 900 1900 2100 MHz HSDPA' }
-- {tech: 4G, bands: '1(2100) 3(1800) 5(850) 7(2600) 8(900) 20(800) 28(700) 38(2600) 40(2300) 41(2500) MHz LTE' }
-peripherals: [Fingerprint reader, Accelerometer, Gyroscope, Proximity sensor, Hall effect sensor, Compass, GPS, NFC]
-ram: 2/3 GB
-recovery_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Power</kbd> until the bootloader appears, then release the buttons.
-  Use the volume buttons to scroll to "RECOVERY" and press <kbd>Power</kbd> to select.
-release: 2015-10-20
-screen: 130 mm (5.0 in)
-screen_ppi: 440
-screen_res: 1920x1080
-screen_tech: Full HD Super AMOLED
-sdcard: up to 2 TB
-soc: Qualcomm MSM8952 Snapdragon 617
-storage: 16/32 GB
-tree: android_device_htc_hiaeuhl
-type: phone
-vendor: HTC
-vendor_short: htc
-versions: [14.1]
-width: 70.8 mm (2.79 in)
-wifi: 802.11 a/b/g/n/ac
diff --git a/_data/devices/hiaeul.yml b/_data/devices/hiaeul.yml
deleted file mode 100644
index c50776b..0000000
--- a/_data/devices/hiaeul.yml
+++ /dev/null
@@ -1,46 +0,0 @@
-architecture: arm64
-battery: {removable: False, capacity: 2150, tech: 'Li-Ion'}
-bluetooth: {spec: '4.1', profiles: [A2DP]}
-cameras:
-- {flash: 'Dual-LED', info: '13 MP'}
-- {flash: '', info: '4 MP'}
-carrier:
-channels: [weekly]
-codename: hiaeul
-cpu: Cortex-A53
-cpu_cores: '8'
-cpu_freq: 4 x 1.5 GHz & 4 x 1.2 GHz
-current_branch: 14.1
-depth: 7.3 mm (0.29 in)
-download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
-gpu: Adreno 405
-height: 145.8 mm (5.74 in)
-image: hiae.png
-install_method: fastboot_htc
-kernel: android_kernel_htc_msm8952
-maintainers: [intervigil]
-models: [2PQ9120]
-name: One A9 (GSM US)
-network:
-- {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
-- {tech: 3G, bands: '850 1700(AWS) 1900 2100 MHz HSDPA' }
-- {tech: 4G, bands: '2(1900) 3(1800) 4(1700) 5(850) 7(2600) 12(700) 13(700) 17(700) 19(850) MHz LTE' }
-peripherals: [Fingerprint reader, Accelerometer, Gyroscope, Proximity sensor, Hall effect sensor, Compass, GPS, NFC]
-ram: 2/3 GB
-recovery_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Power</kbd> until the bootloader appears, then release the buttons.
-  Use the volume buttons to scroll to "RECOVERY" and press <kbd>Power</kbd> to select.
-release: 2015-10-20
-screen: 130 mm (5.0 in)
-screen_ppi: 440
-screen_res: 1920x1080
-screen_tech: Full HD Super AMOLED
-sdcard: up to 2 TB
-soc: Qualcomm MSM8952 Snapdragon 617
-storage: 16/32 GB
-tree: android_device_htc_hiaeul
-type: phone
-vendor: HTC
-vendor_short: htc
-versions: [14.1]
-width: 70.8 mm (2.79 in)
-wifi: 802.11 a/b/g/n/ac
diff --git a/pages/build/hiaeuhl.md b/pages/build/hiaeuhl.md
deleted file mode 100644
index 2e51b81..0000000
--- a/pages/build/hiaeuhl.md
+++ /dev/null
@@ -1,8 +0,0 @@
----
-sidebar: home_sidebar
-title: Build for hiaeuhl
-folder: build
-permalink: /devices/hiaeuhl/build
-device: hiaeuhl
----
-{% include templates/device_build.md %}
diff --git a/pages/build/hiaeul.md b/pages/build/hiaeul.md
deleted file mode 100644
index c971d95..0000000
--- a/pages/build/hiaeul.md
+++ /dev/null
@@ -1,8 +0,0 @@
----
-sidebar: home_sidebar
-title: Build for hiaeul
-folder: build
-permalink: /devices/hiaeul/build
-device: hiaeul
----
-{% include templates/device_build.md %}
diff --git a/pages/info/hiaeuhl.md b/pages/info/hiaeuhl.md
deleted file mode 100644
index 8240b9b..0000000
--- a/pages/info/hiaeuhl.md
+++ /dev/null
@@ -1,9 +0,0 @@
----
-sidebar: home_sidebar
-title: Info about hiaeuhl
-folder: info
-layout: deviceinfo
-permalink: /devices/hiaeuhl/
-device: hiaeuhl
----
-{% include templates/device_info.md %}
diff --git a/pages/info/hiaeul.md b/pages/info/hiaeul.md
deleted file mode 100644
index 2aefffb..0000000
--- a/pages/info/hiaeul.md
+++ /dev/null
@@ -1,9 +0,0 @@
----
-sidebar: home_sidebar
-title: Info about hiaeul
-folder: info
-layout: deviceinfo
-permalink: /devices/hiaeul/
-device: hiaeul
----
-{% include templates/device_info.md %}
diff --git a/pages/install/hiaeuhl.md b/pages/install/hiaeuhl.md
deleted file mode 100644
index 64f57c3..0000000
--- a/pages/install/hiaeuhl.md
+++ /dev/null
@@ -1,8 +0,0 @@
----
-sidebar: home_sidebar
-title: Install LineageOS on hiaeuhl
-folder: install
-permalink: /devices/hiaeuhl/install
-device: hiaeuhl
----
-{% include templates/device_install.md %}
diff --git a/pages/install/hiaeul.md b/pages/install/hiaeul.md
deleted file mode 100644
index 5541ad7..0000000
--- a/pages/install/hiaeul.md
+++ /dev/null
@@ -1,8 +0,0 @@
----
-sidebar: home_sidebar
-title: Install LineageOS on hiaeul
-folder: install
-permalink: /devices/hiaeul/install
-device: hiaeul
----
-{% include templates/device_install.md %}
-- 
2.17.1

